### ISSUE TITLE
[Request] Config Options for Materials/Liquids

### ISSUE THREAD

**Issue by unpure** [McJty/RFTools#113]
Hey there.
I'm currently rebuilding my modpack that runs for my community.
Everyone loves it to build dimensions to geta world he likes.
For some reasons most members of the community hate these materials.
Even with the need of huge amounts of power its way to easy for a long time map.
Because of that it would be awesome to see a config option to disable all of these.
Everything like ores, netherquartz, glowstone and everything else thats "rare" on mining.
The same config option of course for all that liquids. (Maybe only Lava and Water stay.)
If this is not possible, how about a simple config option in the dimlets.cfg for every dimlet?
For now it seems like there would be only options for the rarity and power costs.
Just a single option with true/false to completly disable the dimlet.

Sorry for my bad english, i hope its not to hard to understand.

**Comment by McJty** [comment]
In dimlets.cfg you can blacklist dimlets already. Add a section like this:

knowndimlets {
    I:dimlet.Liquid.cryotheum=-1
    I:dimlet.Material.tile.blockDiamond=-1
}

Do that for every liquid and material that you want to blacklist.
You can also ban the liquids and materials of an entire mod:

knowndimlets {
    I:dimlet.Liquid.cryotheum=-1
    I:dimlet.Material.tile.blockDiamond=-1
    B:modban.Liquid.ThermalFoundation=true
}

Is that ok for your question?

**Comment by unpure** [comment]
Requires a little bit of work in a big modpack but that will work, thank you! :P 
"knowndimlets" sounds more like something the mod itself will fill.
Maybe you could add a little bit more info about this into the notes above this section.
This will prevent future requests/questions about this! :) 
Thank you for your fast answer!

**Comment by McJty** [comment]
Depending on the version of rftools that you have there should be a 'dimletsettings' section either in main.cfg (older versions) or else in dimlets.cfg. That contains all the names. Another way is to do /rftdim dumprarity in game.

And I agree that this must be documented better. I have plans to do that :-)

**Comment by unpure** [comment]
I'm currently rebuilding this pack and therefor using the latest version of RFTools.
I tried to disable all the dimlets with the "knowndimlets" function.
It seems to work fine, but for some reasons there are many dimlets missing.
I've added everything i want to disable thats listed in "dimletsettings" into "knowndimlets".
There are still about 57 liquid and 17 material dimlets that are NOT listed in "dimletsettings".
I also tried the "/rftdim dumprarity" but this only prints something like e.g. 
"Id:402    Key:Lardite.molten"
How would i convert this to work in "knowndimlets"?
Is there any way to get the correct name from the dimlets ingame?
I sorted all dimlets i want to remove that are not listed into a chest so i dont lose them.

Sorry for my bad english :/
